src/nvim/mouse.c

```diff
@@ -1496,7 +1489,7 @@ int jump_to_mouse(int flags, bool *inclusive, int which_button)
       }
     }
   }
-
+foldclick:;
```

Comment:
I think there should be a blank line above this label.
```suggestion

foldclick:;
```